experimental/origin-dapp2/public/dev.html

```diff
@@ -33,5 +33,17 @@
     localStorage.ognNetwork = 'localhost'
     localStorage.metaMaskEnabled = 'true'
   </script>
+  <script async src="https://www.googletagmanager.com/gtag/js?id=UA-106384880-2"></script>
```

Comment:
I think we can remove from dev.html as that only gets used for local development

Reply:
Makes sense. We can still check that the analytics code is working by looking at `window.dataLayer` variable.

Reply:
I've removed this.